Makefile

```diff
@@ -28,10 +28,10 @@ test_lint:
 # for more than once in Travis. Change `make test` in .travis.yml
 # to `make test_multi`.
 test_multi:
-	cd network; \
+	cd services/skipchain; \
```

Comment:
possible, but the `test_multi` I used had sometimes just these four lines, sometimes 2 or 3 times 4 lines, there were lines with `-v`, ... It's just a placeholder so I don't have to spend hours (or even days) re-writing those 4 lines.